examples/plot_variogram_estimation_modelling.py

```diff
@@ -86,8 +74,7 @@
 # large grid data in `scikit-gstat <https://mmaelicke.github.io/scikit-gstat/index.html>`_, which are encapsulated
 # conveniently by :func:`xdem.spatialstats.sample_empirical_variogram`:
 
-df = xdem.spatialstats.sample_empirical_variogram(
-    values=dh.data, gsd=dh.res[0], subsample=300, n_variograms=10, random_state=42)
+df = xdem.spatialstats.sample_empirical_variogram(values=dh.data, gsd=dh.res[0], subsample=100, n_variograms=10, random_state=42)
 
 # %%
 # *Note: in this example, we add a* ``random_state`` *argument to yield a reproducible random sampling of pixels within
```

Comment:
The comment need to be updated to remove the mention to `runs`.

Reply:
At L 91 there is also an outdated ref to Hugonnet et al (in prep).

Reply:
At L126 "one could ask himself" -> "one could wonder"

Reply:
Wow nice catches haha. I'll have a full read to check also.